packages/react/src/composite/SplitButton/CHANGELOG.md

```
@@ -0,0 +1,14 @@
+# Changelog
+
+The Split Button component combines a primary action button with a dropdown menu of related secondary actions.
+
+## 2025-10-21 `1.0.0`
+
+#### 🎉 New features
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)